Send selection email fixes #49

juopmu opened the pull request:
Pending: Still need to write the e-mail text.

CarolinaGri commented:
![screen shot 2018-01-15 at 21 06 35](https://user-images.githubusercontent.com/15719995/34959736-1d72d614-fa38-11e7-999b-a963f9ea802a.png)

CarolinaGri commented:
Hello,
we are happy to tell you that you are accepted for our beginners workshop happening on 11. & 12.8.2017 YAAAAY!!!!
Here are some very important infos. Please read them carefully!
Installation Get-Together: 11.8.2017 18:30-20:30  Where: Co Up 3rd floor Adalbertstr. 8 10999 Berlin-Kreuzberg  Please install Rails by yourself if you cannot come to the party.  Here is the guide: http://guides.railsgirls.com/install If there are problems with the installation please write us.		 Workshop Day: 12.8.2017 10:00-18:00 Where: HTW Berlin Campus Wilhelminenhof Building G 12459 Berlin  Schedule & Details: http://wp.me/P4oXTv-IQ What to bring:  Laptop, charger, extension cord (if you have one), your own coffee cup and something to share at lunch. Think of lunch as a thank you to our coaches who all volunteer on that day :) We will provide warm and cold drinks for you.
IMPORTANT!	Please confirm that you are coming by writing back until 9.8.2017. Please let us know as soon as possible when you cannot come!!! We had a lot of applications and had to reject over 60 people.
So, if you cannot come, make someone else happy. <3 
Feel free to write me, if you have questions.
See you next weekend.
All the best	
Rails Girls Berlin

juopmu commented:
@porras We modified the confirmed method in the applications controller and added a confirmed_too_late view so that the link won't work 5 days after the selection email was sent. Do you think that could work?

porras commented on app/controllers/applications_controller.rb:
```diff
@@ -42,6 +46,17 @@ def select
   end
 
+  def confirm
+    if Time.now > sent_on.5.days.from_now
```
There are two things here that wouldn't work:

1. What's `sent_on`? If it's supposed to be a column, we forgot to add the migration
2. Even if `sent_on` existed (and I guess it would be a `Date` or `Time`), I'm relatively sure it wouldn't have the `5` method (because that method name is illegal, it would be impossible to define 😂). We should discuss what are we trying to achieve here and find out how to do it 😸 

Side note: even if we don't have a specific test for this feature (maybe we should add it), our old tests told us about `2.`! https://travis-ci.org/rubymonsters/rgbapp/builds/331981354#L752
